### DISCUSSION TITLE
Jetson AGX Orin Bluetooth connection to Irobot create 3 problems

### DISCUSSION THREAD

**Discussion by valdivj** [iRobotEducation/create3_docs#408] [Questions]
### How are you connecting to your Create 3?

Bluetooth (Python SDK)

### Computer(s) Model(s) and Operating System(s)

jetson AGX orin ubuntu 20

### Which version of ROS 2 is installed on your computer?

Galactic

### Which firmware version is installed on your robot?

G.4.5

### Which RMW is your robot running?

CycloneDDS

### Is the robot connected to a network? If so, what is the network type?

home network

### Are there multiple Create 3 robots connected to your network?

None

### Is multicast enabled?

No

### What is the Adapter Board's USB/BLE Toggle currently switched to?

Bluetooth (default)

### Describe your question.

When I try to pair or connect to the I robot create 3 using bluetooth it will connect wait 20 seconds then disconnect. Other bluetooth devices have no problem pairing and connecting.

When I use the "blueman-manager"  I get this error:
"blueman-manager 14.18.44 ERROR    Manager:201 error_handler:  Authentication Failed"

When I use "bluetoothd --debug I get this error:
"D-Bus setup failed: Connection ":1.212" is not allowed to own the service "org.bluez" due to security policies in the configuration file"

Any Ideas.

**Comment by shamlian** [comment]
The Create 3 robot is a Bluetooth LE device. If you are trying to pair as a Bluetooth Classic device, that will fail.

**Reply by valdivj** [reply]
I have set "controllermode = dual"
in 
/etc/bluetooth/main.conf

**Reply by shamlian** [reply]
Irrespective of this setting, the Create 3 robot does not do LE bonding. Have you tried running one of the examples in the [iRobot Edu Python SDK](https://github.com/iRobotEducation/irobot-edu-python-sdk/tree/main) or via the web at python.irobot.com (with a browser that supports web-bluetooth like Chrome)?

**Reply by valdivj** [reply]
Yes
 it still has the same connectivity issue.
As a side note it works when I use my windows PC

**Reply by shamlian** [reply]
When you say, "it," do you mean python.irobot.com? There shouldn't be a need to pair with the Windows OS's GUI.

**Reply by shamlian** [reply]
I'm sorry to say I don't know anything about Jetson AGX Orin, so I'm sorry to say I won't be much further help. I can tell you that this SDK _does_ work on my machine natively running Ubuntu, and also on a Raspberry Pi 4 running Raspberry Pi OS, both without additional configuration, so I don't know of any technical reason why it shouldn't work on other similar devices.

**Reply by valdivj** [reply]
interesting
That is good to know.
Thanks for the help

**Reply by shamlian** [reply]
Sure thing; I'm sorry I couldn't help more. Maybe someone with a Jetson will see this and have some ideas.
